fuzz/fuzz_common_code.c

```diff
@@ -1,13 +1,39 @@
 
 #include "fuzz_common_code.h"
 
+
+#ifdef ENABLE_MEM_ALLOC_FAILURES
+
+int mem_alloc_state = 0;
```

Comment:
static? =}

Reply:
No. It is used by the fuzzer themselves
```
ivan@ivan-Latitude-E6540:~/svnrepos/nDPI(fuzz-config)$ git grep mem_alloc_state
[..]
fuzz/fuzz_config.cpp:  mem_alloc_state = size;
fuzz/fuzz_ndpi_reader.c:  mem_alloc_state = Size;
```
Do you prefer some kind of helper like `set_mem_alloc_state()`?

Reply:
Probably a "cleaner" method of localizing global variables.

Reply:
Done!